ext/pdo/pdo_sql_parser.re

```diff
@@ -151,6 +151,7 @@ PDO_API int pdo_parse_params(pdo_stmt_t *stmt, const char *inquery, size_t inque
 	/* did the query make sense to me? */
 	if (query_type == (PDO_PLACEHOLDER_NAMED|PDO_PLACEHOLDER_POSITIONAL)) {
 		/* they mixed both types; punt */
+		/* TODO Always Error */
```

Comment:
I'm inclined to say "no" for the errors in this file. This should be going through the same error reporting as other invalid queries.